src/React.re

```diff
@@ -1,15 +1,33 @@
+[@text "React bindings for ReasonML"];
```

Comment:
Due to a bug in Reason 3.6 (which Jordan fixed recently), floating doc comments `/** */` i.e. ones which are not attached to any structure item, are incorrectly printed as `[@doc "..."]` instead of the correct attribute which is `[@text "..."]`. So we need to directly use `[@text "..."]` for these.